packages/components/src/FormField/FormField.tsx

```diff
@@ -89,15 +94,31 @@ export function FormField(props: FormFieldProps) {
         name={controlledName}
         rules={{ ...validations }}
         defaultValue={value ?? defaultValue ?? ""}
+        // eslint-disable-next-line max-statements
         render={({
           onChange: onControllerChange,
           onBlur: onControllerBlur,
           name: controllerName,
           ...rest
         }) => {
-          const fieldClasses = classnames(styles.formField, {
-            [styles.select]: type === "select",
-          });
+          const fieldClasses = classnames(
+            styles.formField,
+            styles.affix,
+            { [styles.select]: type === "select" },
+            {
+              [styles.withPrefix]: !prefixIcon != !prefixText,
+            },
+            {
+              [styles.withDoublePrefix]: prefixIcon && prefixText,
+            },
+
+            {
+              [styles.withPostfix]: !postfixIcon != !postfixText,
+            },
+            {
+              [styles.withDoublePostfix]: postfixIcon && postfixText,
+            },
```

Comment:
This can be one object, there is no need to create a new object for each check.

```ts
{ 
  [styles.select]: type === "select",
  [styles.withPrefix]: !prefixIcon != !prefixText,
  [styles.withDoublePrefix]: prefixIcon && prefixText,
  [styles.withPostfix]: !postfixIcon != !postfixText,
  [styles.withDoublePostfix]: postfixIcon && postfixText,
}
```

Reply:
👍🏻